datasets/tedlium2.py

```diff
@@ -214,3 +214,66 @@ def load_stm_data(self, stm_file):
                 text = " ".join(tokens[6:])
                 data.append([rec_name, channel, spk_name, start, end, gender, text])
         return data
+
+
+class CreateTEDLIUM2BlissCorpusJobV2(CreateTEDLIUM2BlissCorpusJob):
+    """
+    It has the same logic as CreateTEDLIUM2BlissCorpusJob but the corpus name is distinct for each partition
+    """
+
+    def __init__(self, corpus_folders):
```

Comment:
```suggestion
    def __init__(self, corpus_folders: Dict[str, tk.Path]):
```
Also please add the import at the top please, can't properly in review. Also documentation (even though its not in the other job) would be nice.

Reply:
I will delete the `init` definition as suggested by @michelwi